homeassistant/strings.json

```diff
@@ -34,7 +34,9 @@
         "url": "URL",
         "usb_path": "USB Device Path",
         "access_token": "Access Token",
-        "api_key": "API Key"
+        "api_key": "API Key",
+        "longitude": "Longitude",
+        "latitude": "Latitude"
```

Comment:
When we add common keys for items that are already translated it's good to rename the keys in lokalise for the existing translations too. Someone with admin access in lokalise needs to do that.

Reply:
Du we know who has admin access to do that?

Reply:
Commenting here so we don't forget.

Reply:
I just know Paulus but there should be more admins.